content/1_ModuleOne/11_Foreward.ee.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
We need to remove the .ee.md - but this will be useful for a separate branch that we have.

Reply:
Spoke with @eugenemu, going to leave this in for now with explicit guidance to remove the Event Engine language from the config.toml if it is not applicable to the workshop